### DISCUSSION TITLE
free(): invalid pointer - Abort (core dumped)

### DISCUSSION THREAD

**Discussion by psiddire** [YosysHQ/yosys#3996] [Q&A]
I am synthesizing a design and I get a message which doesn't indicate the error correctly.

All I see is the following message:

free(): invalid pointer
Abort (core dumped)

Any ideas on how to debug it are appreciated.

**Comment by nakengelhardt** [comment]
The first thing to look at is where in the flow this is happening. What are the last messages in the log before that? In what pass is the crash happening? Are you using any plugins or is this a default flow? Does the design get any warnings when using `verilator --lint-only`?
